target/linux/generic/backport-5.15/850-v5.17-0007-PCI-aardvark-Mask-all-interrupts-when-unbinding-driv.patch

```diff
@@ -18,9 +18,9 @@ Signed-off-by: Lorenzo Pieralisi <lorenzo.pieralisi@arm.com>
 
 --- a/drivers/pci/controller/pci-aardvark.c
 +++ b/drivers/pci/controller/pci-aardvark.c
-@@ -1693,6 +1693,27 @@ static int advk_pcie_remove(struct platf
- 	val &= ~(PCI_COMMAND_IO | PCI_COMMAND_MEMORY | PCI_COMMAND_MASTER);
- 	advk_writel(pcie, val, PCIE_CORE_CMD_STATUS_REG);
+@@ -1878,6 +1878,27 @@ static int advk_pcie_remove(struct platf
+ 	/* Remove IRQ handler */
+ 	irq_set_chained_handler_and_data(pcie->irq, NULL, NULL);
  
 +	/* Disable MSI */
```

Comment:
Please remove this patch, it was applied upstream, see https://github.com/gregkh/linux/commit/55e38c42051dc27224c5d27a5d81a8b85c324d0a